client/src/views/users/UserContextMenu.vue

```diff
@@ -52,6 +52,10 @@ export enum UserAction {
   Revoke,
   Details
 }
+export enum UserType {
```

Comment:
No need for an enum, a user is either revoked or not, a simple boolean is enough.